grass7/raster/r.mapcalc.tiled/r.mapcalc.tiled.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+############################################################################
+#
+# MODULE:	r.mapcalc.tiled
+# AUTHOR(S):	Moritz Lennert
+#
+# PURPOSE:	Run r.mapcalc over tiles of the input map
+#               to allow parallel processing
+# COPYRIGHT:	(C) 1997-2018 by the GRASS Development Team
+#
+#		This program is free software under the GNU General Public
+#		License (>=v2). Read the file COPYING that comes with GRASS
+#		for details.
+#############################################################################
+
+#%Module
+#% description: Runs r.mapcalc in parallel over tiles
+#% keyword: raster
+#% keyword: mapcalc
+#% keyword: tiling
+#%end
+#
+#%option
+#% key: expression
+#% type: string
+#% description: Expression to send to r.mapcalc
+#% required: yes
+#%end
+#
+#%option G_OPT_R_OUTPUT
+#% description: Name of raster output map resulting from expression
+#% key: output
+#% required : no
+#%end
+#
+#%option
+#% key: tile_width
+#% type: integer
+#% description: Width of tiles
+#% answer: 1000
+#% required: yes
+#%end
+#
+#%option
+#% key: tile_height
+#% type: integer
+#% description: Height of tiles
+#% answer: 1000
+#% required: yes
+#%end
+#
+#%option
+#% key: overlap
+#% type: integer
+#% description: Overlap between tiles
```

Comment:
```suggestion
#% description: Overlap of tiles
```
Reason: parameter description as in r.tile (i18N compatibility)